phpBB/develop/create_search_index.php

```diff
@@ -132,7 +132,7 @@
 
 $search->tidy();
 
-add_log('admin', 'LOG_SEARCH_INDEX_CREATED', $search_name);
+$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_SEARCH_INDEX_CREATED', time(), array($search_name));
```

Comment:
```php
		if ($log_time == false)
		{
			$log_time = time();
		}
```
Should be false, also there should be 3 equals ;)
So mind fixing the interface and the implementation?